src/python/module/z5py/group.py

```diff
@@ -228,3 +228,39 @@ def require_dataset(self, name, shape,
         return Dataset._require_dataset(path, shape, dtype, chunks,
                                         n_threads, self.is_zarr, self._internal_mode,
                                         **kwargs)
+
+    def visititems(self, func):
+        """ Recursively visit names and objects in this group.
+
+        You supply a callable (function, method or callable object); it
+        will be called exactly once for each link in this group and every
+        group below it. Your callable must conform to the signature:
+
+            func(<member name>, <object>) => <None or return value>
+
+        Returning None continues iteration, returning anything else stops
+        and immediately returns that value from the visit method.  No
+        particular order of iteration within groups is guranteed.
+
+        calls the function @param func with the found items, appended to @param path
+
+        Example:
+
+        # Get a list of all datasets in the file
+        >>> mylist = []
+        >>> def func(name, obj):
+        ...     if isinstance(obj, Dataset):
+        ...         mylist.append(name)
+        ...
+        >>> f = File('foo.n5')
+        >>> f.visititems(func)
+        """
+        for item in self.items():
```

Comment:
Really minor style point - you can unpack `item` here to make its usage more clear later:

    for name, obj in self.items():
        if isinstance(obj, Group):
            # etc